pyproject.toml

```diff
@@ -89,11 +89,9 @@ webbpsf = "pytest_plugin.webbpsf_plugin"
 [project.scripts]
 roman_static_preview = "romancal.scripts.static_preview:command"
 okify_regtests = "romancal.scripts.okify_regtests:main"
-schema_editor = "romancal.scripts.schema_editor:main"
-schemadoc = "romancal.scripts.schemadoc:main"
```

Comment:
Neither of these exist and attempting to run `schema_editor` or `schemadoc` after installing romancal produces an error.